src/wp-includes/blocks/latest-posts.php

```diff
@@ -50,8 +50,16 @@ function render_block_core_latest_posts( $attributes ) {
 
 	$filter_latest_posts_excerpt_more = static function( $more ) use ( $attributes ) {
 		$use_excerpt = 'excerpt' === $attributes['displayPostContentRadio'];
-		/* translators: %1$s is a URL to a post, excerpt truncation character, default … */
-		return $use_excerpt ? sprintf( __( ' … <a href="%1$s" rel="noopener noreferrer">Read more</a>' ), esc_url( get_permalink() ) ) : $more;
+		if ( $use_excerpt ) {
+			/* translators: %1$s is a URL to a post, excerpt truncation character, default … */
```

Comment:
```suggestion
```
The translators comment is no longer needed as only `'Read more'` is translated here, and it includes no tokens to be explained. 🙂